urlwatch

```diff
@@ -388,10 +427,26 @@ if __name__ == '__main__':
         short_summary += '\n\n\n'
         if not options.quiet:
             print short_summary
+
+	if(options.pushover):
+          from chump import Application
```

Comment:
Also here, don't do the import here, but do it on the top of the file. Also, if you already used `import chump` above, why not use `chump.Application`?